tests/frontier/opcodes/test_call_code_gas.py

```diff
@@ -24,6 +24,22 @@
 )
 from ethereum_test_tools.vm.opcode import Opcodes as Op
 
+"""
+CALL gas breakdowns: (https://www.evm.codes/#f1)
+memory_exp_cost + code_exec_cost + address_access_cost + positive_value_cost + empty_account_cost
+= 0 + 0 + 2600 + 9000 + 0 = 11600
+"""
+CALL_INSUFFICIENT_GAS = 0x2D50  # 11600
```

Comment:
I would not call this `CALL_INSUFFICIENT_GAS`, for this opcode this amount should be exactly the right amount to execute this specific opcode. However, in this test we need some extra gas (7* push opcode gas) to init the stack.

Also, `empty_account_cost` should be nonzero (I think it is 25000)

Reply:
Thanks, that makes sense! I agree the naming is not ideal. 
I like the option you specified below using:
`CALL_SUFFICIENT_GAS -1`

Reply:
For CALL this value should be 36600, see: https://github.com/ethereumjs/ethereumjs-monorepo/blob/330a3d74da69ed8b0b8b720797138d9192ac65cf/packages/evm/test/runCall.spec.ts#L729

Reply:
Also, small-ish note: this gas var is only correct for a range of hardforks (I think Berlin and all hardforks after that), since the gas costs of CALL have changed a lot (including other semantics such as "warm/cold account access" per EIP 2929)

Reply:
> When filling with geth initially I tried to use 36600: [2c64526#diff-b376d818c93291d54fc1fefc658bade757f8b83e03384718a5048461a6f067a4R32](https://github.com/acolytec3/execution-spec-tests/commit/2c64526206c5a3c7af7fc25595cbddf4b360d8d1#diff-b376d818c93291d54fc1fefc658bade757f8b83e03384718a5048461a6f067a4R32)
> 
> However, the SSTORE from the CALL was 1. I.e 36600 was sufficient. Maybe there could be an issue in geth?

I will check now :smile: